src/pages/groupComparison/GroupComparisonMutationMapper.tsx

```diff
@@ -46,8 +76,119 @@ export default class GroupComparisonMutationMapper extends MutationMapper<
+                    this.props.store.indexedMyVariantInfoAnnotations
+                }
+                oncoKbData={this.props.store.oncoKbData}
+                usingPublicOncoKbInstance={
+                    this.props.store.usingPublicOncoKbInstance
+                }
+                mergeOncoKbIcons={this.props.mergeOncoKbIcons}
+                onOncoKbIconToggle={this.props.onOncoKbIconToggle}
+                civicGenes={this.props.store.civicGenes}
+                civicVariants={this.props.store.civicVariants}
+                enableOncoKb={this.props.enableOncoKb}
+                enableFunctionalImpact={this.props.enableGenomeNexus}
+                enableHotspot={this.props.enableHotspot}
+                enableMyCancerGenome={this.props.enableMyCancerGenome}
+                enableCivic={this.props.enableCivic}
+                generateGenomeNexusHgvsgUrl={
+                    this.props.generateGenomeNexusHgvsgUrl
+                }
+                isCanonicalTranscript={this.props.store.isCanonicalTranscript}
+                selectedTranscriptId={this.props.store.activeTranscript.result}
+                columnVisibility={this.props.columnVisibility}
+                storeColumnVisibility={this.props.storeColumnVisibility}
+                namespaceColumns={this.props.store.namespaceColumnConfig}
+                columns={this.columns}
+                profiledPatientCountsByGroup={
+                    this.props.groupToProfiledPatientCounts
+                }
+                groups={this.props.groups}
+                formatPaginationStatusText={this.formatPaginationStatusText}
+                showTotalMutationCountsInCountHeader={true}
+                sampleSet={this.props.sampleSet}
+                customControls={this.tableCustomControls}
+                getRowDataByProteinChange={
+                    this.rowDataByProteinChangeForEnrichedGroups
+                }
+                initialSortColumn={'p-Value'}
+                initialSortDirection={'asc'}
+            />
+        );
+    }
+
+    @computed get rowDataByProteinChange() {
+        let dataStore = this.props.store.dataStore as MutationMapperDataStore;
+        let mutationsByProteinChange = _.values(
+            _.groupBy(_.flatten(dataStore.allData), d => d.proteinChange)
+        );
+        return getProteinChangeToMutationRowData(
+            mutationsByProteinChange,
+            this.mutatedCountsByProteinChangeForGroup,
+            this.props.groupToProfiledPatientCounts,
+            this.props.groups
+        );
+    }
+
+    @autobind
+    protected rowDataByProteinChangeForEnrichedGroups():
+        | {
+              [proteinChange: string]: ComparisonMutationsRow;
+          }
+        | {} {
+        return _.pickBy(this.rowDataByProteinChange, v => {
+            return this.significanceFilter
+                ? this._enrichedGroups.includes(v.enrichedGroup) &&
+                      v.qValue < 0.05
```

Comment:
0.05 should be a constant defined somewhere and we should explain in comment it's meaning.